index.html

```diff
@@ -0,0 +1,34 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <title>Document</title>
+    <link href="style.css" rel="stylesheet" type="text/css">
+</head>
+
+<body>
+    <h1> Resume of Rick from Rick and Morty</h1>
+    <nav>
+        <h2>Experience -<a href="pages/experience.html"> Click me </a></h2>
+        <h2>Bio of Rick -<a href="pages/bio.html">Click me</a></h2>
+    </nav>
+            <p>Please<a href="https://www.youtube.com/watch?v=3gJ1dDUQe2s">click here</a> for a sample portfolio theme song from Rick and Morty</p>
+        <div class="picf">    
+            <img src="images/rick.jpg" alt="Image of Rick" title="Rick's Selfie" class="center" />
+        <h3>Rick and Morty Family Pic</h3>
+            <img src="images/rm.jpg" alt="Image of Rick and Morty" title="Rick's Professional Pic" />
+        <h3>Professional Picture of Rick and Morty</h3>
+            <img src="images/rick1.jpg" alt="k's miltary picture" title="Rick's Miltary Pic" />
+        <h3>Rick's Miltary Exxperience</h3>
+        </div>
```

Comment:
@cliffjadoo I see it now, but initially it is very hard to see. Can you please go back and Indent your code 🔨